.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,164 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
this is a IDE-based config file, we should probably add it to .gitignore?

Reply:
Hm yes. We could probably add the entire codeStyles directory to .gitignore.